docs/2.deploy/20.providers/firebase.md

```diff
@@ -31,271 +23,7 @@ You can integrate with this provider using [zero configuration](/deploy/#zero-co
     - Choose a region.
     - Import a GitHub repository (you’ll need to link your GitHub account).
     - Configure deployment settings (project root directory and branch), and enable automatic rollouts.
-    - Choose a unique ID for deployment.
-4. Wait for the first release to complete.
-
--->
-
-## Firebase hosting
```

Comment:
Continuing https://github.com/nitrojs/nitro/pull/3215#discussion_r1991023005 
@pi0

Reply:
Adding deprecated notice seems good idea (note that current docs will soon be moved to v2.nitro.build as legacy)

Reply:
Added the old docs with a new deprecated note! I'm also going to reopen the old pr and copy over these changes too. That way the changes will be reflected in the new docs too